extensions/wikia/Email/i18n/Welcome.i18n.php

```diff
@@ -156,23 +156,23 @@
 );
 
 $messages['nl'] = array(
-	'emailext-welcome-subject' => 'Welcome to Fandom, $1!',
-	'emailext-welcome-summary' => 'Welcome to Fandom! You are now part of the world’s largest collection of entertainment communities. Share your knowledge, discuss opinions, and discover great content.',
-	'emailext-welcome-summary-extended' => "So what's next?",
+	'emailext-welcome-subject' => 'Welcome to Wikia, $1!',
```

Comment:
It seems this is not used anywhere:
<img width="888" alt="screenshot 2016-11-07 10 01 02" src="https://cloud.githubusercontent.com/assets/20514390/20051790/1b532342-a4d1-11e6-8562-e2535b7bbca5.png">